spec/MongoStorageAdapter.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const MongoStorageAdapter = require('../src/Adapters/Storage/Mongo/MongoStorageAdapter');
+const MongoStorageAdapter = require('../src/Adapters/Storage/Mongo/MongoStorageAdapter').MongoStorageAdapter;
```

Comment:
the newer airbnb wants:

`const {MongoStorageAdapter} = require('../src/Adapters/Storage/Mongo/MongoStorageAdapter');`

all the cool kids are doing it.

Reply:
lol :) most definitely :)